openeo/rest/datacube.py

```diff
@@ -523,34 +523,48 @@ def reduce_tiles_over_time(self, code: str, runtime="Python", version="latest"):
         """
         return self.reduce_temporal_udf(code=code, runtime=runtime, version=version)
 
-    def apply(self, process: str, data_argument='x') -> 'DataCube':
+    def apply(self, process: Union[PGNode, str], data_argument='x') -> 'DataCube':
         # TODO #125 allow more complex sub-process-graphs?
+
+        arguments = {"data": self._pg}
+
+        if self._api_version.at_least("1.0.0"):
+            if isinstance(process, str):
+                arguments["process"] = {"process_graph": PGNode(
+                        process_id=process,
+                        arguments={data_argument: {"from_parameter": "x"}})}
+            else:
+                arguments["process"] = {"process_graph": process}
+        else:
+            if isinstance(process, str):
+                arguments["process_graph"] = PGNode(
+                        process_id=process,
+                        arguments={data_argument: {"from_parameter": "x"}})
```

Comment:
for 0.4 it has to be `from_argument` I think

Reply:
Yes true, done in latest commit